tools/pylint-extensions/azure-pylint-guidelines-checker/pylint_guidelines_checker.py

```diff
@@ -1449,6 +1469,13 @@ def check_return(self, node):
                 has_return = True
             if line.startswith("rtype"):
                 has_rtype = True
+                try:
+                    if line.split("rtype")[1] and ":class" in line.split("rtype")[1]:
+                            self.add_message(
+                                msgid="docstring-type-do-not-use-class", args=(line.split("rtype")[1]), node=node, confidence=None
+                            )
+                except:
+                    pass
```

Comment:
Do we need to check for :class: in :raises:?

Reply:
Just double checked: ` you can use the :class: role to link to a class object in the Raises section, as long as the class is defined in the same module or imported from another module.`